docs/portal/image-registry/managing-repositories-under-a-project.md

```diff
@@ -5,8 +5,8 @@ A project can have one or more repositories and a repository can have one or mor
 The following procedure describes how to view repositories under a project, how to change a repository's description, and how to delete a repository.
 
 1. In the left pane, click **Image Registry > Registry Projects**.
-2. On the Registry Projects page, click the name of a project. The Edit page appears. This page has two sections:<ul><li>**Basic information**  shows read-only information about the selected project.</ul></li><ul><li>**Repositories** shows the repositories associated with the selected project. A field at the top left allows you to filter the repositories shown.</ul></li>
-![null](</docs/resources/images/registry/edit-repository.png>)
-3. To change a repository's description, click the pencil icon in the **Description** column. When the field opens, change the description, and then click the check mark.
-![null](</docs/resources/images/registry/repository-description.png>)
-4. To delete a repository, click its **Actions** menu, and then select **Delete**. When prompted, click **Yes, delete!**
\ No newline at end of file
+2. On the Registry Projects page, click the name of a project. The detail page appears. This page has two sections:<ul><li>**Basic information**  shows read-only information about the selected project.</ul></li><ul><li>**Repositories** shows the repositories associated with the selected project. A field at the top left allows you to filter the repositories shown.</ul></li>
+<p align=center><img src="/docs/resources/images/registry/edit-repository.png" width="600"></p>
+3. To change a repository's description, click the pencil icon in the <strong>Description</strong> column. When the field opens, change the description, and then click the check mark.
+<p align=center><img src="/docs/resources/images/registry/repository-description.png" width="600"></p>
+4. To delete a repository, click its <strong>Actions</strong> menu, and then select **Delete**. When prompted, click <strong>Yes, delete!</strong>
```

Comment:
**Delete** is not properly rendered on page preview. I notice that you sometimes use <strong> label pair, and sometimes use double asteriks. Why?

Reply:
Some times, using double asterisks following a GUI snap shot prevents the double asterisks from bolding text, as you saw in this case. In these cases, I use the <strong></strong> elements. Sorry to have missed this one.